modules/accredited_representative_portal/spec/requests/accredited_representative_portal/v0/in_progress_forms_controller_spec.rb

```diff
@@ -0,0 +1,100 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+require_relative '../../../support/authentication'
+
+RSpec.describe AccreditedRepresentativePortal::V0::InProgressFormsController, type: :request do
+  let(:representative_user) { create(:representative_user) }
+  let(:expected_response_body) do
+    {
+      'data' => {
+        'id' => in_progress_form.id.to_s,
+        'type' => 'in_progress_forms',
+        'attributes' => {
+          'formId' => in_progress_form.form_id,
+          'createdAt' => in_progress_form.created_at,
+          'updatedAt' => in_progress_form.updated_at,
+          'metadata' => in_progress_form.metadata
+        }
+      }
+    }
+  end
```

Comment:
Because this is a simple CRUD resource I am pretty tempted to turn this request spec into one test that only modifies things with a series of hardcoded requests and assertions about the response in terms of those hardcoded requests. Something like:
```
GET -> DELETE -> PUT -> GET -> PUT -> GET -> DELETE -> GET
```
We can talk about it--I don't think it's much code. I'm not horribly opposed to keeping using factories as setup and each example making one request.